include/cxx_project.sh

```diff
@@ -67,9 +67,8 @@ function make_project
 	else
 		if [ "$4" == "true" ]
 		then
-			rm -rf "$PROJECT_NAME"
+			rm -rf "$PROJECT_NAME/*"
 		fi
-		mkdir "$PROJECT_NAME"
```

Comment:
I just do not want to destroy the folder, just its content. So I do not need to recreate the directory.